src/utils/helpers.js

```diff
@@ -43,8 +43,13 @@ export const toUpperCaseFirstLetter = (string) =>
  * @return {String} format phone number
  */
 export const formatPhoneNumber = (phoneNumberString) => {
+  const lastPartLength = phoneNumberString.length - 6;
+  const reg = `^(\\d{3})(\\d{3})(\\d{${lastPartLength}})$`;
+  const regex = new RegExp(reg);
+
   const cleaned = ` ${phoneNumberString}`.replace(/\D/g, '');
-  const match = cleaned.match(/^(\d{3})(\d{3})(\d{3})$/);
+  const match = cleaned.match(regex);
+
   if (match) {
     return `(${match[1]})  ${match[2]} - ${match[3]} `;
```

Comment:
Change to descriptive naming please, what does match[1] mean?
example: 
```suggestion
    return `(${CountryCode})  ${AreaCode} - ${LineNumber} `;
```